bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/communication/RotelConnector.java

```diff
@@ -458,6 +464,15 @@ public void sendCommand(RotelCommand cmd, @Nullable Integer value) throws RotelE
                                     messageStr += String.format("-%02d", -value);
                                 }
                                 break;
+                            case BALANCE_SET:
+                                if (value == 0) {
+                                    messageStr += "000";
+                                } else if (value > 0) {
+                                    messageStr += String.format("r%02d", value);
+                                } else {
+                                    messageStr += String.format("l%02d", -value);
```

Comment:
Reading the specification, the l should be uppercase.

Reply:
see above